R/get_current_providers.R

```diff
@@ -0,0 +1,41 @@
+#' Fetch leaflet providers from Leaflet.js.
+#' @export
+#'
+#' @return List of \code{providers}, \code{providers.details}
```

Comment:
Now that the docs are markdown, I believe these should be back-ticks now instead of `\code` 

```
#' @return List of `providers`, `providers.details`
```